### ISSUE TITLE
role="presentation" not supported on SVG element

### ISSUE THREAD

**Issue by jhensley** [dequelabs/axe-core#3082]
Product: axe-core

Expectation: Allow `role="presentation"` on `<svg>` elements

Actual: `role="presentation"` not seen as a valid role on `<svg>` elements

Motivation: Based on the WAI specs, any role should be valid on an `<svg>` element

`<svg role="presentation" alt="foo"></svg>`
axe-core version: 2.3.0

[![Edit axe-core react (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/axe-core-react-forked-vjpho?fontsize=14&hidenavigation=1&theme=dark)

**Comment by straker** [comment]
Thanks for the issue. Our code is out-of-date with the latest version of the HTML in ARIA spec. [Back when we first took from it](https://github.com/w3c/html-aria/blob/d29561f67c3a6cf702807a0241c6171282c08b83/index.html#L2380-L2394) SVG was only allowed `application`, `document`, and `img`. [The latest spec allows all roles](https://www.w3.org/TR/html-aria/#el-svg)

**Comment by jhensley** [comment]
Makes sense, I only discovered it when upgrading to axe-core 4.3.0; guessing some other check (perhaps this: https://github.com/dequelabs/axe-core/pull/2989) is picking it up or something.
